docs/tutorial/using-checkbox/test-output.rst

```diff
@@ -0,0 +1,80 @@
+.. _test_output:
+
+============================
+Test output
+============================
+
+When Checkbox runs the tests, it generates a series of outputs, which include the ID and category of the tests, along with the results of each them (if a test ID fails, the cause for the failure would be displayed). 
+
+The test output can be generally divided into the following sections:
+
+- **Hardware Manifest**:
```

Comment:
This is a job title, it is true that the manifest job is run every time, but it is not "special". It is just another job.